tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -61,82 +47,6 @@ var _ = Describe("odo devfile create command tests", func() {
 		})
 	})
 
-	Context("When executing odo create with devfile component type and component name arguments", func() {
-		It("should successfully create the devfile component with valid component name", func() {
-			helper.CmdShouldPass("odo", "create", "java-openliberty", cmpName)
-		})
-
-		It("should fail to create the devfile component with component name that contains invalid character", func() {
```

Comment:
I have removed some of the integration tests which are purely conditional one. IMO we should cover this in unit test. I have also tracked it in issue https://github.com/openshift/odo/issues/4052